app/assets/javascripts/govuk_publishing_components/components/checkboxes.js

```diff
@@ -23,6 +23,26 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         }
       });
 
+      $(scope).find('input[type=checkbox]').on('change', function(e) {
+        var $checkbox = $(e.target);
+        var category = $checkbox.data("track-category");
+        if (typeof category !== "undefined") {
+          var isChecked = $checkbox.is(":checked");
+          var uncheckTrackCategory = $checkbox.data("uncheck-track-category");
+          if (!isChecked && typeof uncheckTrackCategory !== "undefined") {
+            category = uncheckTrackCategory;
+          }
+          var action = $checkbox.data("track-action");
+          var options = $checkbox.data("track-options");
+          if(typeof options !== 'object' || options === null){
+            options = {};
+          }
+          options['value'] = $checkbox.data("track-value");
+          options['label'] = $checkbox.data("track-label");
+          GOVUK.analytics.trackEvent(category, action, options);
```

Comment:
We probably want to wrap this in `if (GOVUK.analytics && GOVUK.analytics.trackEvent) {` like we do for other tracking JS, as otherwise it errors when GOVUK.analytics is not available.